pandas/core/indexing.py

```diff
@@ -1795,7 +1795,7 @@ def error():
                     error()
                 raise
             except:
-                error()
```

Comment:
why did this need changin?

Reply:
The old way hides the cause of the exception.

For example if the exception is in ``ax.contains`` you won't see where the actual error was in the traceback, as Python thinks the exeption is **in** ``error``, while in reality is is somet´where else, hence the need to reraise.

Thought on closer inspection, just removing the two lines ``except: error()`` would be even clearer than reraising.